<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>613</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/*background:url(:/new/pics/1614372377_41-p-svetlii-fon-bumaga-69.jpg)
background:rgb(255, 242, 225);;*/
background:url(:/new/pics/slightly-blotted-clean-pale-stucco-pattern.jpg);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px  solid grey;
background: url(:/new/pics/a5fz3O1qK_U.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>251</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background:rgb(255, 171, 172);
border-radius: 10px 10px 10px 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="gridFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 242, 225);
border: 1px  solid white;
margin-left:15px;
font: 700 10pt &quot;Segoe Script&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 242, 225);
border: 1px  solid white;
margin-left:15px;
font: 700 10pt &quot;Segoe Script&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe Script&quot;; color:black;border:none;</string>
        </property>
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe Script&quot;; color:black;border:none;</string>
        </property>
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe Script&quot;; color:black;border:none;</string>
        </property>
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe Script&quot;; color:black;
border:none;</string>
        </property>
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 242, 225);
border: 1px  solid white;
margin-left:15px;
font: 700 10pt &quot;Segoe Script&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 242, 225);
border: 1px  solid white;
margin-left:15px;
font: 700 10pt &quot;Segoe Script&quot;;
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 242, 225);
font: 700 10pt &quot;Segoe Script&quot;;
color:black;
border-radius: 10px 10px 10px 10px;
border: 1px  solid white;
padding:3px;
padding-right:15px;
padding-left:15px;

</string>
        </property>
        <property name="text">
         <string>Обрезать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="gridFrame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 242, 225);
font: 700 10pt &quot;Segoe Script&quot;;
color:black;
border: 1px  solid white;
border-radius: 10px 10px 10px 10px;
padding:2px;</string>
        </property>
        <property name="text">
         <string>Монохром</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSlider" name="MonSlider">
        <property name="styleSheet">
         <string notr="true">background:none;border:none;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="PaletteButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255)) ;
font: 700 11pt &quot;Segoe Script&quot;;
border: 1px  solid white;
border-radius: 10px 10px 10px 10px;
background:rgb(255, 242, 225);</string>
        </property>
        <property name="text">
         <string>Палитра</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>241</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:rgb(255, 171, 172);
border-radius: 10px 10px 10px 10px;

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Colorlbl_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px  solid black;
border-radius: 20px 10px 20px 10px;
background:white;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Colorlbl_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px  solid black;
border-radius: 20px 10px 20px 10px;
background:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Colorlbl_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px  solid black;
border-radius: 20px 10px 20px 10px;
background:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Colorlbl_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px  solid black;
border-radius: 20px 10px 20px 10px;
background:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Colorlbl_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px  solid black;
border-radius: 20px 10px 20px 10px;
background:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Colorlbl_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px  solid black;
border-radius: 20px 10px 20px 10px;
background:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>191</width>
        <height>130</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">Color:rgb(170, 0, 0);
font: 700 12pt &quot;Segoe Script&quot;;border:none;</string>
         </property>
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="Red_Slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="Green_Slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">Color:rgb(0, 170, 0);
font: 700 12pt &quot;Segoe Script&quot;;border:none;</string>
         </property>
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="Blue_Slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">Color:rgb(0, 0, 255);
font: 700 12pt &quot;Segoe Script&quot;;</string>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>160</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="originButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 242, 225);
font: 700 10pt &quot;Segoe Script&quot;;
color:black;
border: 1px  solid white;
border-radius: 10px 10px 10px 10px;
padding: 2px;</string>
         </property>
         <property name="text">
          <string>Предпросмотр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Back_origButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(255, 242, 225);
font: 700 10pt &quot;Segoe Script&quot;;
color:black;
border: 1px  solid white;
border-radius: 10px 10px 10px 10px;
padding: 2px;</string>
         </property>
         <property name="text">
          <string>К оригиналу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background:rgb(255, 171, 172);
border-radius: 10px 10px 10px 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Размер изображения</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Ширина:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_width">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>Высота:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_height">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Segoe Script&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Script</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Script&quot;;
background:rgb(255, 242, 225);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Segoe Script&quot;;
background:rgb(255, 242, 225);</string>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 242, 225);</string>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/new/pics/open.png</normaloff>:/new/pics/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Sources.qrc">
     <normaloff>:/new/pics/save.png</normaloff>:/new/pics/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Sources.qrc"/>
 </resources>
 <connections/>
</ui>
